packages/input-iban/src/formatters.js

```diff
@@ -7,5 +7,8 @@ import { friendlyFormatIBAN } from '@bundled-es-modules/ibantools';
  * @return {string} formatted value
  */
 export function formatIBAN(modelValue) {
+  if (modelValue === '') {
```

Comment:
If this is a bug because ibantools doesn't handle `''` properly in their formatter, maybe nice to make an issue for it on their repo :)?

Reply:
Good suggestion. It's a bug in ibantools indeed. Maybe also add a comment about it (defensive code because of ibantools) @gerjanvangeest

Reply:
https://github.com/Simplify/ibantools/issues/15